pkg/bootstrap/stage.go

```diff
@@ -166,7 +166,18 @@ func extractFromDir(dir, prefix string, img v1.Image, imgName string) error {
 	if err := extract(imgName, tempDir, prefix, r); err != nil {
 		return err
 	}
-	return os.Rename(tempDir, dir)
+	if err := os.Rename(tempDir, dir); err != nil {
+		fileInfo, err := ioutil.ReadDir(tempDir)
```

Comment:
```suggestion
		files, err := ioutil.ReadDir(tempDir)
```

`fileInfo` is a little misleading as `ioutil.ReadDir` returns a slice of FileInfo's.